addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/AbstractHomematicGateway.java

```diff
@@ -117,6 +115,8 @@ public AbstractHomematicGateway(String id, HomematicConfig config, HomematicGate
     public void initialize() throws IOException {
         logger.debug("Initializing gateway with id '{}'", id);
 
+        scheduler = config.getThreadPoolFactory().getScheduledPool("homematicGateway");
```

Comment:
You won't need this.
Whereever in your code you want to schedule a job, simply use

```
ThreadPoolManager.getScheduledPool("homematicGateway");
```